src/morale.h

```diff
@@ -22,6 +22,8 @@ enum morale_type : int
     MORALE_FEELING_GOOD,
     MORALE_SUPPORT,
     MORALE_PHOTOS,
+    MORALE_HAIRCUT,
+    MORALE_SHAVE,
```

Comment:
Technically, the entries in this enum also have to stay in order as they are used in the saves. But morale is temporary anyway, getting this out of oder will probably not have a lasting effect, and is most likely only used for displaying it.